src/LocalFileWriter.js

```diff
@@ -0,0 +1,66 @@
+const fs = require('fs')
+const uuid = require('node-uuid')
+const path = require('path')
+const Stream = require('stream')
+const { callbackify, promisify } = require('util')
+const Settings = require('settings-sharelatex')
```

Comment:
This is where Settings come from in other modules:
```suggestion
const Settings = require('./Settings')
```